src/acktest/bootstrapping/sqs.py

```diff
@@ -0,0 +1,61 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from re import S
+import boto3
+
+from dataclasses import dataclass, field
+
+from . import Bootstrappable
+from .. import resources
+
+@dataclass
+class Queue(Bootstrappable):
+    # Inputs
+    name_prefix: str
+    policy: str = ""
+    
+    # Outputs
+    arn: str = field(init=False)
+    
+    @property
+    def sqs_client(self):
+        return boto3.client("sqs", region_name=self.region)
```

Comment:
i do not see self.region field initialized anywhere

Reply:
It's inherited from `Bootstrappable` class